aviary/mission/flops_based/ode/mission_ODE.py

```diff
@@ -83,28 +83,27 @@ def setup(self):
                         context='mission'),
             promotes_inputs=['*'],
             promotes_outputs=['*'])
+
         self.add_subsystem(
             name='atmosphere',
-            subsys=USatm1976Comp(num_nodes=nn),
-            promotes_inputs=[('h', Dynamic.Mission.ALTITUDE)],
-            promotes_outputs=[
-                ('sos', Dynamic.Mission.SPEED_OF_SOUND), ('rho', Dynamic.Mission.DENSITY),
-                ('temp', Dynamic.Mission.TEMPERATURE), ('pres', Dynamic.Mission.STATIC_PRESSURE)])
+            subsys=Atmosphere(num_nodes=nn, input_speed_type=SpeedType.MACH),
+            promotes=['*'],
+        )
 
         # add an execcomp to compute velocity based off mach and sos
-        self.add_subsystem(
-            name='velocity_comp',
-            subsys=om.ExecComp(
-                'velocity = mach * sos',
-                mach={'units': 'unitless', 'shape': (nn,)},
-                sos={'units': 'm/s', 'shape': (nn,)},
-                velocity={'units': 'm/s', 'shape': (nn,)},
-                has_diag_partials=True,
-            ),
-            promotes_inputs=[('mach', Dynamic.Mission.MACH),
-                             ('sos', Dynamic.Mission.SPEED_OF_SOUND)],
-            promotes_outputs=[('velocity', Dynamic.Mission.VELOCITY)],
-        )
+        # self.add_subsystem(
```

Comment:
Why is the speed comp commented out?

Reply:
This calc is done by the flight conditions, so this is now redundant. If you're asking why it's commented out instead of deleting it, I must have missed that in my cleanup pass now that _most_ tests are passing now - or at least tests directly relevant to checking the ODE in isolation